godot-core/src/registry/signal/connect_builder.rs

```diff
@@ -25,231 +29,51 @@ use crate::registry::signal::{SignalReceiver, TypedSignal};
 // - Pack multiple types together into "type lists", i.e. custom structs carrying the type state. For a user, this would appear as one type,
 // - which could also be #[doc(hidden)]. However, this may make the trait resolution more complex and worsen error messages, so not done now.
 ///
-/// # Builder stages
-///
-/// The builder API has a well-defined flow and is separated in stages. In each stage, you have certain builder methods available that you can
-/// or must call, before advancing to the next stage. Check the instructions.
-///
-/// ## Stage 1 (required)
-/// Choose one:
-/// - [`function`][Self::function]: Connect a global/associated function or a closure.
-/// - [`object_self`][Self::object_self]: If you want to connect a method (in stage 2), running on the same object as the signal.
-/// - [`object`][Self::object]: If you want to connect a method, running on a separate object.
-///
-/// ## Stage 2 (conditional)
-/// Required iff _(if and only if)_ `object_self` or `object` was called in stage 1.
-/// - [`method_mut`][Self::method_mut]: Connect a `&mut self` method.
-/// - [`method_immut`][Self::method_immut]: Connect a `&self` method.
-///
-/// ## Stage 3
+/// # Customization
 /// All these methods are optional, and they can be combined.
 // Use HTML link due to conditional compilation; renders badly if target symbol is unavailable.
-/// - [`sync`](#method.sync): If the signal connection should be callable across threads.  \
-///   Requires `Send` + `Sync` bounds on the provided function/method, and is only available for the `experimental-threads` Cargo feature.
-/// - [`name`][Self::name]: Name of the `Callable` (for debug purposes).  \
+/// - [`name()`][Self::name]: Name of the `Callable` (for debug purposes).  \
 ///   If not specified, the Rust function name is used. This is typically a good default, but not very readable for closures.
 /// - [`flags`][Self::flags]: Provide one or multiple [`ConnectFlags`][crate::classes::object::ConnectFlags], possibly combined with bitwise OR.
```

Comment:
Minor thing, my mistake
```suggestion
/// - [`flags()`][Self::flags]: Provide one or multiple [`ConnectFlags`][crate::classes::object::ConnectFlags], possibly combined with bitwise OR.
```